app/src/main/java/com/hieuminh/chatapp/presenter/contract/ChatHomeContact.kt

```diff
@@ -0,0 +1,16 @@
+package com.hieuminh.chatapp.presenter.contract
+
+import com.hieuminh.chatapp.model.MessageModel
+import com.hieuminh.chatapp.presenter.BasePresenter
+
+interface ChatHomeContact {
+
+    interface Presenter : BasePresenter {
+        fun onGetValue()
```

Comment:
```suggestion
        fun getValue()
```
Method name in presenter should be started with a verb